### ISSUE TITLE
Relation members update 

### ISSUE THREAD

**Issue by jbelien** [RobinLinde/ESN-edit#5]
If you edit a relation, does the `name:etymology:wikidata` tag is also applied to the relation members (just the ways of the street, of course) ?

In EqualStreetNames data process, I use all the relation member that have `role=outer` or `role=street` but it might probably need some checks more strict to be sure you can update the members too (could be manual confirmation in your UI).

**Comment by RobinLinde** [comment]
I believe it should currently update only the relation itself, though it might be worth to consider updating member of `associatedStreet` relations (with `role=street`). I didn't really think about that, since there aren't that many in the Netherlands. Searching did lead to finding one weird one: https://www.openstreetmap.org/relation/6816114

**Comment by jbelien** [comment]
Yeah, I know that "associatedStreet" is not used a lot worldwide but most of the streets of Brussels are "associatedStreet" relations (and I think it's also used a lot in Germany).

Definitely not something urgent, just convenient (since for Brussels, I would have to manually update the ways after updating the relation through your tool). 😄
